release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2936,3 +2936,28 @@ link:https://access.redhat.com/solutions/6585021[{product-title} 4.8.24 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-8-25"]
+=== RHBA-2021:5209 - {product-title} 4.8.25 bug fix and security update
+
+Issued: 2022-01-05
+
+{product-title} release 4.8.25, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:5209[RHBA-2021:5209] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2021:5208[RHSA-2021:5208] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6620441[{product-title} 4.8.25 container image list]
+
+[id="ocp-4-8-25-bug-fixes"]
+==== Bug fixes
+
+* Previously, `ovnkube-node` and `ovnkube-master` pods failed to start, when the config file had an unknown field or section. Consequently, it failed to update OVN-Kubernetes. This update makes OVN-Kubernetes resilient for an unknown field in the config file and logs a warning instead of exiting if such a field was found. As a result, OVN-Kubernetes updates do not fail if config file contains an unknown field or section. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2022171[*BZ#2022171*])
```

Comment:
Small changes: 

* Previously, `ovnkube-node` and `ovnkube-master` pods failed to start when the config file had an unknown field or section. Consequently, OVN-Kubernetes would not update. With this update, OVN-Kubernetes no longer exits if unknown fields are found in the config file. Instead, a warning is logged. As a result, OVN-Kubernetes updates no longer fail if the config file contains an unknown field or section. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2022171[*BZ#2022171*])